### ISSUE TITLE
Web Server Startup failed

### ISSUE THREAD

**Issue by dbwest** [dnikolayev/sonarqube-mysql-windows-docker#2]
`Execution of migration step #1701 'Clean orphans from USER_ROLES' failed`

**Comment by dnikolayev** [comment]
Hello, David

Some more details will help a lot :)
Which Windows, what Docker version, what SonarQube version, etc. Unfortunately I have no telepathic abilities to reproduce the problem ;)

Also, please provide the way you run.

Thanks!

**Comment by dnikolayev** [comment]
As no response came, I assume that the problem was resolved. Closing.

**Comment by arturohernandez10** [comment]
I got the same error.  #1701 'Clean orphans from USER_ROLES': failure. I am using windows 10 pro (10.0.18362). And for Docker it's  19.03.8. Architecture windows/amd64.

Here is the image I used:
REPOSITORY                                 TAG                 IMAGE ID            CREATED             SIZE
dnikolayev/sonarqube-mysql-windows         latest              85d3b574d406        23 months ago       3.83GB

This is my docker command

docker run --name sonar --memory 2g -it -p 9030:9000  ^
    -v C:\Projects\TiffsTreats\sonarqube\mysql:C:\MySQL\data ^
    -v C:\Projects\TiffsTreats\sonarqube\extensions:C:\sonarqube\extensions ^
    dnikolayev/sonarqube-mysql-windows:latest

I realize this is an old image, But I am running windows, and the official docker image is for linux.
